be/api/po-bl/src/main/java/cz/covid/po/api/bl/service/CodebookServiceImpl.java

```diff
@@ -50,9 +62,21 @@ public void init() {
     }
 
     @Getter
-    @AllArgsConstructor
     private static class CodebookRegistration {
+
+        public CodebookRegistration(Class<? extends CodebookItemBase> codebookClass, Class<? extends CodebookItemRepository> repositoryClass){
+            this.codebookClass = codebookClass;
+            this.repositoryClass = repositoryClass;
+        }
         private Class<? extends CodebookItemBase> codebookClass;
         private Class<? extends CodebookItemRepository> repositoryClass;
+
+        public Class<? extends CodebookItemBase> getCodebookClass() {
+            return codebookClass;
+        }
+
+        public Class<? extends CodebookItemRepository> getRepositoryClass() {
+            return repositoryClass;
+        }
```

Comment:
Tohle je vsechno lombok - potreba nakonfigurovat IDE, kdyz tak...

https://projectlombok.org/

Reply:
Na konfiguraci IDE kouknu pro Lombok kouknu, potreboval jsem to rychle hodit do java8, aby to slo deploynout. Java11 java containery jsou, ale neslo to  deplyounut. Ted se to deployuje, az to fixnem, tak si tam vratite java 11